### PR TITLE
OpenRPC spec sync

### PR DESCRIPTION
**Pull request by aoudiamoncef**
* [x] document all added functions
* [x] try in sandbox /simulation/labnet
* [x] unit tests on the added/changed features
  * [x] make tests compile
  * [x] make tests pass 
* [x] add logs allowing easy debugging in case the changes caused problems
* [x] if the API has changed, update the API specification

### CODE REVIEW

**Comment by damip** [comment]
@aoudiamoncef  can you also check if we can improve the playground ?
* make it act directly on a testnet node
* add an "intro" paragraph with a few typical usage examples
* make it easier to perform a call (check how this "method" bug can be bypassed)

File: README.md [hunk 1]
```diff
@@ -25,6 +25,9 @@ An easy-to-read blog post introduction with videos is written
 We are now releasing the **Massa testnet** in this GitHub repository,
 with its explorer available at <https://massa.net/testnet>.
 
+You could interact with Massa JSON-RPC public API on [OpenRPC Playground](https://playground.open-rpc.org/?schemaUrl=https://test.massa.net/api/v2&schema=https://raw.githubusercontent.com/massalabs/massa/main/openrpc.json&uiSchema\[appBar\]\[ui:input\]=false&uiSchema\[appBar\]\[ui:inputPlaceholder\]=Enter+Massa+JSON-RPC+server+URL&uiSchema\[appBar\]\[ui:logoUrl\]=https://massa.net/favicons/favicon.ico&uiSchema\[appBar\]\[ui:splitView\]=false&uiSchema\[appBar\]\[ui:darkMode\]=false&uiSchema\[appBar\]\[ui:title\]=Massa&uiSchema\[appBar\]\[ui:examplesDropdown\]=false&uiSchema\[methods\]\[ui:defaultExpanded\]=false&uiSchema\[methods\]\[ui:methodPlugins\]=true&uiSchema\[params\]\[ui:defaultExpanded\]=false)which is an equivalent of SwaggerUI for OpenAPI specification.
```
Comment (damip) [inline README.md:28]: could -> can

 for OpenAPI specification =>  for OpenAPI specifications

File: openrpc.json [hunk 2]
```diff
@@ -0,0 +1,2331 @@
+{
+    "openrpc": "1.2.4",
+    "info": {
+        "title": "Massa OpenRPC Specification",
+        "version": "TEST.15.1",
+        "description": "Massa OpenRPC Specification document. Find more informations on https://docs.massa.net/en/latest/technical-doc/api.html",
```
Comment (damip) [inline openrpc.json:6]: informations => information

**Comment by AurelienFT** [comment]
@aoudiamoncef can we merge this in prepare-testnet-16 ?

**Comment by aoudiamoncef** [comment]
> @aoudiamoncef can we merge this in prepare-testnet-16 ?

I think that it's ready for `main`

**Pull request merged**